.github/workflows/build.yml

```diff
@@ -0,0 +1,229 @@
+            git config --global user.name 'swodlr-ui bot'
+            git config --global user.email 'swodlr-ui@noreply.github.com'
+            git commit -am "/version ${{ env.THE_VERSION }}"
+            git push
+          fi
+
+      - name: Push Tag
+        if: |
+          github.ref == 'refs/heads/develop' ||
+          github.ref == 'refs/heads/main'    ||
+          startsWith(github.ref, 'refs/heads/release')
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+          git tag -f -a "${{ env.THE_VERSION }}" -m "Version ${{ env.THE_VERSION }}"
+          git push origin "${{ env.THE_VERSION }}" --force
+
+      # - name: Publish UMM-T with new version
+      #   uses: podaac/cmr-umm-updater@0.5.0
+      #   if: |
+      #     github.ref == 'refs/heads/main' ||
+      #     startsWith(github.ref, 'refs/heads/release')
+      #   with:
+      #     umm-json: cmr/${{ env.THE_ENV }}_swodlr_cmr_umm_t.json
+      #     provider: 'POCLOUD'
+      #     env: ${{ env.THE_ENV }}
+      #     version: ${{ env.THE_VERSION }}
+      #     timeout: 60
+      #     disable_removal: 'true'
+      #     umm_type: 'umm-t'
+      #     use_associations: 'false'
+      #   env:
+      #     LAUNCHPAD_TOKEN_SIT: ${{secrets.LAUNCHPAD_TOKEN_SIT}}
+      #     LAUNCHPAD_TOKEN_UAT: ${{secrets.LAUNCHPAD_TOKEN_UAT}}
+      #     LAUNCHPAD_TOKEN_OPS: ${{secrets.LAUNCHPAD_TOKEN_OPS}}
+
+      # Setup Node to install and test
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+
+      - name: NPM install & NPM Lint
+        run: |
+          npm install
+          npm run build
+
+      # - name: Make Config Files
+      #   run: |
+      #     cat ./configs/hitideConfig-${{ env.THE_ENV }}.js > ./dist/hitideConfig.js
+      
+      ## Deployment
+      - name: Deploy Env Override
+        if: |
+          github.event.head_commit.message == '/deploy sit' ||
+          github.event.head_commit.message == '/deploy uat'
+        run: |
+          message="${{ github.event.head_commit.message }}"
+          trimmed_message=${message:1}  # Remove leading slash
+          override_env=$(echo "$trimmed_message" | grep -oE '[^[:space:]]+$')
+          override_env_upper=$(echo "$trimmed_message" | awk '{print toupper($NF)}')
+          echo "THE_ENV=${override_env}" >> $GITHUB_ENV
+          echo "TARGET_ENV_UPPERCASE=${override_env_upper}" >> $GITHUB_ENV
+
+      - name: Configure AWS credentials 
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: us-west-2
+          role-session-name: GitHubActions
+          aws-access-key-id: ${{ secrets[format('AWS_ACCESS_KEY_ID_SERVICES_{0}', env.TARGET_ENV_UPPERCASE)] }}
+          aws-secret-access-key: ${{ secrets[format('AWS_SECRET_ACCESS_KEY_SERVICES_{0}', env.TARGET_ENV_UPPERCASE)] }}
+
```

Comment:
We're missing a terraform deploy step here that has to happen before syncing the files to s3. Once the terraform deployment is done, we get the name of the bucket from terraform output using
                                            
`terraform output swodlr-bucket-name`

Where `swodlr-bucket-name` is the name of the `output` defined in `terraform/swodlr-main.tf`

Reply:
It looks like hitide-ui is also missing this step